test/Microsoft.Extensions.Certificates.Generation.Tests/CertificateManagerTests.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.Extensions.Certificates.Generation.Tests
+{
+    public class CertificateManagerTests
+    {
+        [Fact]
+        public void EnsureCreateHttpsCertificate_CreatesACertificate_WhenThereAreNoHttpsCertificates()
+        {
+            // Arrange
+            const string CertificateName = nameof(EnsureCreateHttpsCertificate_CreatesACertificate_WhenThereAreNoHttpsCertificates) + ".cer";
+            var manager = new CertificateManager();
+
+            manager.RemoveAllCertificates(CertificatePurpose.HTTPS, StoreName.My, StoreLocation.CurrentUser);
```

Comment:
😬 what if I have certs I want to keep? IMO it would be better to track certs you've made in the test run and ensure they are cleaned up at the end of the test.

Reply:
Oh hell yes. Naughty.